mtags/src/main/scala/scala/meta/internal/metals/TextEdits.scala

```diff
@@ -22,10 +22,7 @@ object TextEdits {
       val out = new java.lang.StringBuilder()
       positions.foreach { case (edit, pos) =>
         out.append(text, curr, pos.start)
-        edit.getNewText().foreach {
```

Comment:
This was causing some tests to look weird and almost impossible to test fails with tabs in them.